v4-client-js/src/clients/modules/post.ts

```diff
@@ -98,9 +111,13 @@ export class Post {
       zeroFee: boolean,
       gasPrice: GasPrice = GAS_PRICE,
       memo?: string,
+      account?: () => Promise<Account>,
     ): Promise<Uint8Array> {
-      const msgs = await messaging();
-      return this.signTransaction(wallet, msgs, zeroFee, gasPrice, memo);
+      const msgsPromise = await messaging();
+      const accountPromise = account ? (await account()) : this.account(wallet.address!);
```

Comment:
```suggestion
      const msgsPromise = messaging();
      const accountPromise = account ? account() : this.account(wallet.address!);
```
Since we are using promise.all later, we do not need to await here, since the await will unravel the promise.